src/compiler/checker.ts

```diff
@@ -365,6 +365,93 @@ namespace ts {
 
         const argumentsSymbol = createSymbol(SymbolFlags.Property, "arguments" as __String);
         const requireSymbol = createSymbol(SymbolFlags.Property, "require" as __String);
+        const fail = (name: string) => () => Debug.fail("Unit kind expected for " + name)
+        // boolean, bigint, number, string, symbol [literal], boolean, bigint, number, string, symbol, object, union (recursive?!), ..., unknown, never, null, void, undefined, any
+        const ordre: Array<[TypeFlags, (t1: Type, t2: Type) => Comparison]> = [

```

Comment:
order

Reply:
This is a draft, so this is just the name that fell out of my brain and it happened to be in French. (I often use short, intentionally bad names so I'll be forced to change them later.)

Reply:
Got it :)